drivers/i2c/i2c_ll_stm32_v1.c

```diff
@@ -597,29 +597,55 @@ int i2c_stm32_error(const struct device *dev)
 	const struct i2c_stm32_config *cfg = dev->config;
 	struct i2c_stm32_data *data = dev->data;
 	I2C_TypeDef *i2c = cfg->i2c;
-
 #if defined(CONFIG_I2C_TARGET)
```

Comment:
The extra empty line above was fine. Clearly separates common local variables from the extra one used when target mode is enabled. Non-block: feel free to address this comment or not.